src/resp_parser.c

```diff
@@ -150,13 +151,11 @@ static int parseDouble(ReplyParser *parser, void *p_ctx) {
     parser->curr_location = p + 2; /* for \r\n */
     char buf[MAX_LONG_DOUBLE_CHARS + 1];
     size_t len = p - proto - 1;
-    double d;
+    double d = 0;
```

Comment:
We also want to make the change for https://github.com/valkey-io/valkey/blob/unstable/src/util.c#L595. The current code is just for lua parsing.

Reply:
> We also want to make the change for https://github.com/valkey-io/valkey/blob/unstable/src/util.c#L595. 
Let me create a separate PR for it. I saw that strtod invocation. 

> double d = 0;
Are you looking at previous revision of code? I already have this default initialization.

Reply:
That is like the main reason to use it, I think it should be in this PR.

Reply:
Converted all invocations.